vuepress/docs/next/tutorials/create/ms/generate-microservices-and-micro-frontends.md

```diff
@@ -1,81 +1,67 @@
-
-4. Enter "Yes" when prompted with the following overwrite to resolve the conflict:
-
-`Overwrite .gitignore?`
-
-5. You'll be presented with a series of additional prompts pertaining to project configuration. These are echoed below, with the base values for this tutorial in parentheses. Input your preferences, except where a required entry is identified in **bold**. Note that the `Enter` key will select the default option.
-
+3. You'll be presented with a series of prompts pertaining to service generation. These are echoed below, with the base values for this tutorial in parentheses. Input your preferences, except where a required entry is identified in **bold**. Note that the `Enter` key will select the default option.
 
     - `Please provide the project name:` (Up to you)
-
     - `What is the base name of your application?` (Up to you) 
     - `As you are running in a microservice architecture, on which port would like your server to run? It should be unique to avoid port conflicts.` (8081)
     - `What is your default Java package name?` (Up to you)
     - `Which *type* of database would you like to use?` (SQL)
           - If no database is selected you'll be building a stateless microservice, which is a valid choice, but the rest of this tutorial won't work.
-    - `Which *production* database would you like to use?` (PostgreSQL or MySQL)
+    - `Which *production* database would you like to use?` (PostgreSQL)
     - `Which *development* database would you like to use?` (H2 with disk-based persistence)
      - `Which cache do you want to use? (Spring cache abstraction)` (Caffeine (local cache, for a single node))
     - `Do you want to use Hibernate 2nd level cache?` (Yes)
     - `Which other technologies would you like to use?` (**Don't select any other technologies**)
     - `Which BE dependencies do you want to use?` (Dependencies maintained by Entando (entando/entando-bundle-bom))
-    - `What name would you give to the bundle to share on an Entando Component Repository?` (Up to you)
-    - `Which is the organization name to use when publishing the docker image?` (**Enter the name of the organization where you are going to push your Docker image. If you're using your personal Docker hub account enter your username.**) <-- this can be changed later as needed
     - `Would you like to generate micro frontends when creating entities?` (Always)
     - `Would you like to enable internationalization support` (Up to you)
     - `Please choose the native language of the application` (Up to you)
     - `Besides JUnit and Jest, which testing frameworks would you like to use?` (Up to you)
     - `Would you like to install other generators from the JHipster Marketplace?` (No)
 
-6. Next, add an Entity to your microservice and create the corresponding micro frontends. In this tutorial, `Conference` is the name of the entity that will be added to the application.
+4. Enter "Yes" when prompted with the following overwrite to resolve the conflict:
```

Comment:
is this step still needed? (I think it was the case because the cli created a .gitignore file but now we run the bp in a separated folder, it should be different)

Reply:
@avdev4j The prompt was still showing up for me in my last test. We're planning to test one more time when the blueprint is finalized and can update the text to match then.